apps/server/src/modules/user/controller/admin-api-user.controller.ts

```diff
@@ -15,7 +15,7 @@ export class AdminApiUsersController {
 	@ApiOperation({
 		summary: 'create a user together with an account',
 	})
-	async createSchool(@Body() body: AdminApiUserCreateBodyParams): Promise<AdminApiUserCreateResponse> {
+	async createUser(@Body() body: AdminApiUserCreateBodyParams): Promise<AdminApiUserCreateResponse> {
```

Comment:
not part of the ticket. Here we noticed the name of the function was simply a mistake